### DISCUSSION TITLE
New simple tips to Mint 21.3.

### DISCUSSION THREAD

**Discussion by geraldotdsjr** [linuxmint/discussions#266] [Ideas]
Hi, I from Brazil.

Below I showing simple tips to make Linux Mint a perfect software (in my opinion).

![alt-tab-mode](https://github.com/linuxmint/discussions/assets/57339631/142c81b2-5cee-43e7-bc66-e88bf1fd05e0)

Please, consider use the Alt+Tab Only Icons mode, its more beautiful and simple.

![show-desktop-button](https://github.com/linuxmint/discussions/assets/57339631/d6779b94-8fa2-4648-ab89-6d3d1f342a9e)

Please, consider use the previous "Show Desktop Button" position. New format similar to Windows 10 is not a good idea. Mint MATE and XFCE we have the previous format.

![system-sounds](https://github.com/linuxmint/discussions/assets/57339631/c78826ca-b9bc-44d8-99bc-4af9604dbb73)

Cinnamon sounds not beautiful. Only notifications is good (similar this image). I disable all.

![cursor](https://github.com/linuxmint/discussions/assets/57339631/2fb6b7c1-71c8-4f64-9755-d88dae36eef8)

Cursors Adwaita, DMZ or Yaru (best choice), Bibata is modern buts not beautiful and functional.

![home-folders](https://github.com/linuxmint/discussions/assets/57339631/d7e5c0d9-781c-4dc9-ab37-6be66f7d16d8)

Home folder options to choose default or rename to your language (similar to Elementary menu). I change this using ".user-dirs.dirs".

![command-tab](https://github.com/linuxmint/discussions/assets/57339631/d3254ff8-6200-4afe-9c4a-c86c3aaa21d9)

Consider use Command + Tab and Alt + Tab in MacBooks (similar to Elementary). Two options in same time is a good idea (similar to Ubuntu).

**Comment by 100savage** [comment]
Most of the things you would like to change can be modified by the user. I wouldn't think the Mint team would not make changes like this to please a small percentage of people using the system.

**Comment by igorsantos07** [comment]
Alt-tab with icons only is not feasible when you have more than one screen from the same app - which is quite easy with browsers or file managers. Although that could show the title only when there's more than one screen...

Other than that, I agree that you've showed a lot of personal preferences as "your tips". It would be silly if the maintainers changed the defaults based on some random user requesting them, given you can easily change them to your liking - as you did. I assure you people who developed the interface have had some thought on where to put stuff and how to set the defaults lol
